### ISSUE TITLE
Coding-tasks - open for everyone!

### ISSUE THREAD

**Issue by Angamanga** [ushahidi/platform#4750]
Where you would like to improve accessibility in the Ushahidi platform client (web)?
- Look through the platform client and document one or more issue (one accessibility-problem per issue) in GitHub (https://github.com/ushahidi/platform/issues/new/).
- Include [Outreachy Task Submission] in the title.
- Solve the problem and submit a PR!

**Comment by ademolaomosanya** [comment]
would like to work on the task kindly assign me @Angamanga

**Comment by felabel** [comment]
Hi, I added this issue - https://github.com/ushahidi/platform/issues/4751 but there are no labels to tag them to. I am not sure I have access to label issues. this is screenshot of the new issue @Angamanga 

![Image](https://github.com/ushahidi/platform/assets/63490084/961a96fe-b6b9-486b-b3b3-7ecbd2e48df7)

**Comment by Angamanga** [comment]
@felabel You need to add the category in surveys->add a survey (or select an existing one) -> add a category field and select your newly selected category to add to that survey.

**Comment by felabel** [comment]
@Angamanga i don’t really get your comment, are you referring to adding the category to a survey on the web client or being able to label an issue ?

**Comment by Chiemezuo** [comment]
@Angamanga Pardon me for asking, but I just want to be extra sure. Are we doing this for `platform-client` or for `platform-client-mzima` ?
Seeing as it's the latter we installed.

**Comment by kivenalaric** [comment]
Hello, I added this issue - https://github.com/ushahidi/platform/issues/4759  but there are no labels to tag them to. I am not sure I have access to label issues. this is screenshot of the new issue @Angamanga

![Image](https://github.com/ushahidi/platform/assets/89359876/fa2b881c-2d43-48e3-b3e4-1eeb6831139d)

**Comment by Angamanga** [comment]
@felabel Sorry, I was confusing you :D  Add [Outreachy Task Submission] in the title.
@Chiemezuo For the platform-client-mzima 👍 
@kivenalaric You can add [Outreachy Task Submission] in the title instead 👍

**Comment by claire256** [comment]
Hello, I added this issue #4804, the screenshot is added below. Though could not add the “Outreachy Task Submission” label. @Angamanga.

![Image](https://github.com/ushahidi/platform/assets/101701821/9054b373-dbdc-4535-8b36-5260887e7389)

**Comment by jona42-ui** [comment]
> Hello, I added this issue [#4804](https://github.com/ushahidi/platform/issues/4804), the screenshot is added below. Though could not add the “Outreachy Task Submission” label. [@Angamanga](https://github.com/Angamanga).
> 
> ![Image](https://private-user-images.githubusercontent.com/101701821/311749808-9054b373-dbdc-4535-8b36-5260887e7389.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAyNDY2MDQsIm5iZiI6MTcxMDI0NjMwNCwicGF0aCI6Ii8xMDE3MDE4MjEvMzExNzQ5ODA4LTkwNTRiMzczLWRiZGMtNDUzNS04YjM2LTUyNjA4ODdlNzM4OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxMlQxMjI1MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMjg4ZTFhZjcxMzUwN2Q1NWVjM2I0YmFkMjYxYjJmYjgxZDU3MTQ1YWE2MDY3OTgwYmM1NTU2MWE5NDljMTc3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.u9EQQvVHIG7ekE8ztjvR4RkcD1mR_iVtQlG37ggGqu0)

its meant to be included in the issue 
title

**Comment by claire256** [comment]
@jona42-ui Let me add it, thank you.

**Comment by blessdamy** [comment]
Hello @Angamanga I created an issue for an accessibility-problem I discovered on #4875 . I would love to fix it, but I don't know how to go about it. Your guidance will be highly appreciated. Thank you😇

![Issue11](https://github.com/ushahidi/platform/assets/111387281/44ab2f1f-0e29-4523-9851-29c2be9825e5)
